Source/APIClient/APIClient.swift

```diff
@@ -2,16 +2,32 @@ import Alamofire
 
 final class APIClient {
 
+    enum HeaderType {
+        case chainpointJson
+        
+        func value() -> HTTPHeaders {
+            switch self {
+            case .chainpointJson:
+                return [
+                    "Content-Type":"application/json",
```

Comment:
When we create a `POST` request with Alamofire, this header added by default. Is it not enough to append the below value?